snorkel/labeling/model/label_model.py

```diff
@@ -654,6 +651,7 @@ def train_model(
         self._set_class_balance(class_balance, Y_dev)
         self._set_constants(L_train)
         self._create_tree()
+        self.L_train = L_train
```

Comment:
Let's compute coverage here via snorkel.labeling.analysis rather than carrying around the matrix